src/utilities/modules/vaccines.js

```diff
@@ -0,0 +1,41 @@
+import { formattedDifferenceBetweenDates, formatExposureRange } from '../formatters';
+
+/**
+ * Method for creating data for BreachTable rows.
+ * Accepts an array (Example below) of objects,
+ * each having a breach property which contains
+ * a realm results of sensorLog objects.
+ * @param {Array} data Example below
+ * [ {
+ *     breach: [ sensorLogs ],
+ *     ...
+ *   },
+ *    ...
+ * ]
+ */
+export const setBreachData = (data = []) => {
+  const dataCopy = [];
+  data.forEach(dataSet => {
+    const { breach } = dataSet;
+    // A breach must have a minimum of 3 sensorlogs to be considered
+    // a breach.
+    if (!breach || breach.length < 4) return;
```

Comment:
i don't think you need to restrict to 3 sensorLogs ?

Reply:
hm, there is one case where we won't have 3 `sensorLogs` - if the first two `sensorLogs` are breached, and the one after isn't.

(I just realized I need to add some more logic here to find the first sensor log to use)

The surrounding `sensorLogs` of a breach will also be in this array, so will have `4` in most cases, `3` if it is the very last breach that has occured (min. 2 sensor logs which have been breached?).

I needed to set a minimum number of `SensorLogs` to calculate values (or set these values as null if there just *happened* to be 1 `SensorLog` in this array.) and if there really was less than 3 (other than the first ever 2 `sensorLogs`, something must have gone wrong. 

Can reduce this to two, or set values to null if < 3?

Reply:
> hm, there is one case where we won't have 3 sensorLogs - if the first two sensorLogs are breached, and the one after isn't.

data here comes from extractBreaches, which appends sensorLogs at both ends, i guess it's possible if we don't actually have logs at both ends, but still that's enough for us ?